pkg/kubeadm/cmd/manual.go

```diff
@@ -0,0 +1,195 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/renstrom/dedent"
+	"github.com/spf13/cobra"
+
+	kubeadmapi "k8s.io/kubernetes/pkg/kubeadm/api"
+	kubemaster "k8s.io/kubernetes/pkg/kubeadm/master"
+	kubenode "k8s.io/kubernetes/pkg/kubeadm/node"
+	kubeadmutil "k8s.io/kubernetes/pkg/kubeadm/util"
+	netutil "k8s.io/kubernetes/pkg/util/net"
+	// TODO: cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+)
+
+var (
+	manual_done_msgf = dedent.Dedent(`
+		Master initialization complete:
```

Comment:
For some of the issues it seems reasonable to get a starting point in and list these in an issue as things to discuss and clean up.  It'll help keep the discussions scoped and easier to digest.

For more design/UX oriented (which commands should we have) we should lay out the scenarios and go over the right way to express that.  You or Luke or me or someone could pull together a strawman and we could go over the details in the weekly meeting.